src/main/java/org/opensearch/agent/tools/PPLTool.java

```diff
@@ -373,4 +405,20 @@ private String parseOutput(String llmOutput, String indexName) {
         return ppl;
     }
 
+    private String loadDefaultPrompt(PPLModelType pplModelType) {
```

Comment:
Now we read the resource every time when invoke creating PPLTool. It would be more efficient if we only read them once and store the prompts at static fields and

Reply:
Already move it to static.